### PR TITLE
Last changes from Tag 4.2.0 4201

### PR DESCRIPTION
**Pull request by frankeros**
Hi team,
this PR includes the last changes about index pattern fix, message for agent deployments and remove Node warning for insecure flag

### CODE REVIEW

File: server/lib/api-interceptor.ts [hunk 1]
```diff
@@ -12,8 +12,13 @@
...
 import axios, { AxiosResponse } from 'axios';
 import { ManageHosts } from './manage-hosts';
+import https from 'https';
 
-process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
+const httpsAgent = new https.Agent({
+  rejectUnauthorized: false,  
+});
+
+const _axios = axios.create({ httpsAgent });
```
Comment (gabiwassan) [inline server/lib/api-interceptor.ts:21]: 🙌🏼

**Review by gabiwassan — APPROVED** [review]
LGTM 🚀

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by eze9252 — APPROVED** [review]
LGTM!

**Pull request merged**